backend/src/openbeheer/utils/decorators.py

```diff
@@ -0,0 +1,37 @@
+from functools import wraps
+from typing import Callable
+from rest_framework.response import Response
+from requests.exceptions import ConnectionError
+
+from openbeheer.types._open_beheer import ExternalServiceError
+
+
+def handle_service_errors(func: Callable) -> Callable:
```

Comment:
Does this gel well with pyright? Or will it erase types and infer Callable[Unknown]?
I remember doing some type-fu to get well typed decorators.

Reply:
It does indeed infer with Unknown :S I'll try to fix it :ninja: